README.md

```diff
@@ -1,5 +1,35 @@
-# Example notebooks
+# nteract example notebooks
 
-This repository has collections of notebooks to try out on Binder, organized by language.
+**Go ahead and give these notebooks a try.**
 
+This repository has collections of **nteract** notebooks to try out on Binder. Click the link of an individual example or the language heading for a bundle of notebooks.
+
+## [Python](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fpython)
+
+* [The Notable Nteract Notebook](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fpython%2Fintro.ipynb)
+* [Try out Vega through Altair](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fpython%2Faltair.ipynb)
+* [Make Progress (Bars) with Updatable Outputs](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fpython%2Fdisplay-updates.ipynb)
+* [Glean the Download Statistics for nteract Desktop](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fpython%2download-stats.ipynb)
+* [Explore Your World with GeoJSON](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fpython%2Fgeojson.ipynb)
 * [Exploring Happiness Across Countries](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fpython%2Fhappiness.ipynb)
+* [Meet Markdown and Math](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fpython%2Fmarkdown-regression-testing.ipynb)
+* [Debug nteract's Super Secret SetIn Models](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fpython%2Fmodel-debug.ipynb)
+* [Go from Pandas to GeoJSON](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fpython%2Fpandas-to-geojson.ipynb)
+* [Gallery of Plotly Plots](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fpython%2Fplotly.ipynb)
+* [Pandas Table Schema Advancements](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fpython%2Ftable-with-schema.ipynb)
+* [VDOMmable Updates with Emojis](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fpython%2Fvdom.ipynb)
+
+## [R](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fr)
+
+* [Raw Plotly](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fr%2Fplotlyr.ipynb)
+* [VegaLite for R](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fr%2Fvegalite-for-r.ipynb)
+
+## [node.js](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fnode.js)
+
+* [Exploring Custom Revival with JSON.parse - Immutable Revival](https://mybinder.org/v2/gh/nteract/examples/master?urlpath=%2Fnteract%2Fedit%2Fnode.js%2Fimmutable-revival.ipynb)
```

Comment:
This one is a bit of a cheat since we don't have a node kernel installed. Should check to see if it's available on conda I suppose